### ISSUE TITLE
Settings

### ISSUE THREAD

**Issue by LarryMoe11** [Xinyuan-LilyGO/T-01C3#1]
What are the settings that need to be used? I cant figure it out.

**Comment by LilyGO** [comment]
What are your problems now?

**Comment by tally0highwayman** [comment]
Can you please add a schematic showing how to program this module. 

UPDATE: Here is the wiring connections that were recommended to me. 

VCC = 3.3 volts
RST = pulled high (resistor to VCC)
GPIO2 = pulled high (resistor to VCC)
GND = GND
GPIO9 = = pulled low (resistor to GND)
RX = Programmer TX
TX = Programmer RX

**Comment by LilyGO** [comment]
Hello.
[Reference](https://github.com/Xinyuan-LilyGO/T-01C3#quick-start)

**Comment by raz0rsedge1975** [comment]
In your Reference is nothing written about the setup needed for programming the module.

**Comment by LarryMoe11** [comment]
it does, what do you need help with @raz0rsedge1975 ? (this thread is 2 years old and the problem was eventually resolved, I just forgot to respond)

**Comment by raz0rsedge1975** [comment]
I meant LilyGO, he/she refered to this github page, quick-start. But could not find a howto for programming the module. have a Ch430 USB2TTL and need to program des ESP32-C3 module's wifi...

**Comment by LarryMoe11** [comment]
The syntax may be slightly different, but this tutorial gets the general idea across, most of the commands should be the same in this library. https://deepbluembedded.com/esp32-connect-to-wifi-network-arduino/#esp32-connect-to-wifi-network-example

**Comment by raz0rsedge1975** [comment]
Yeah thanks, the coding is clear, I got stuck wiring the module up for programming. You wrote resistor to VCC(3.3V) and resistor to ground, question is what resistor(?ohms) should I use?

**Comment by LarryMoe11** [comment]
you only NEED a resistor if your power source is 5v, its still recommended at 3.3v though (im not sure why, just going off what ive seen). I would use somewhere between a 1-10kohm resistor. if you find your module isnt powering up, or is shutting down intermittantly, reduce the resistance.

**Comment by evlo** [comment]
How do I put this into firmware upload mode?
